### DISCUSSION TITLE
tutorials/networking/http_request_class

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#4] [User-contributed notes]
# tutorials/networking/http_request_class

Why use HTTP?: HTTP requests are useful to communicate with web servers and other non-Godot programs. Compared to Godot's other networking features (like High-level multiplayer), HTTP requests have...

https://docs.godotengine.org/en/latest/tutorials/networking/http_request_class.html

**Comment by Swarkin** [comment]
I made an `AwaitableHTTPRequest` Node to make HTTP requests more convenient and just in one line:
https://github.com/Swarkin/Godot-AwaitableHTTPRequest

**Reply by bruno-pmakers** [reply]
That's really helpful.
